dist/style.css

```diff
@@ -292,5 +295,8 @@ a {
 }
 
 hr {
-  width: 50%;
+  display:block;
+  background-image: url('OrnamentalDivider.svg');
```

Comment:
**For relative paths:**

`./` means "starting at the current directory"
`../` means "go backwards one directory"
`../../` means "go backwards two directories"

Please put this file back in the `./dist/assets/other` directory, and change the path here to:

`./dist/assets/other/OrnamentalDividier.svg`